lib/streamlit/__init__.py

```diff
@@ -507,3 +509,12 @@ def stop():
 
     """
     raise StopException()
+
+
+def experimental_rerun():
+    """Reruns the streamlit application.
+
+    When run outside of Streamlit this will raise an Exception.
+    """
```

Comment:
Can we change this to:

```
    """Rerun the script immediately.

    When `st.experimental_rerun` is called, the script is halted - no more
    statements will be run, and the script will be queued to re-run from the
    top.

    If this function is called outside of Streamlit, it will raise an
    Exception.
    """
```

Reply:
Can do. Thanks :).